src/algebra/algebra/restrict_scalars.lean

```diff
@@ -96,17 +98,34 @@ The preferred way of setting this up is `[module R M] [module S M] [is_scalar_to
 instance : module R (restrict_scalars R S M) :=
 module.comp_hom M (algebra_map R S)
 
+instance : is_scalar_tower R S (restrict_scalars R S M) :=
```

Comment:
This instance is about the "forbidden" `restrict_scalars.module_orig` instance, right? Indicating that in a docstring would be good.